build error

Hi !!
I got the following error when I tried to contribute to this repo and I did `npm run build`.

```
src/widlprocess.ts:178:59 - error TS2531: Object is possibly 'null'.

178         if (extAttr.name === "NamedConstructor" && typeof extAttr.rhs.value === "string") {
                                                              ~~~~~~~~~~~


src/widlprocess.ts:180:23 - error TS2531: Object is possibly 'null'.

180                 name: extAttr.rhs.value,
                          ~~~~~~~~~~~


error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/src/widlprocess.ts
+++ b/src/widlprocess.ts
@@ -175,7 +175,7 @@ function getConstructor(extAttrs: webidl2.ExtendedAttributes[], parent: string) 
 
 function getNamedConstructor(extAttrs: webidl2.ExtendedAttributes[], parent: string): Browser.NamedConstructor | undefined {
     for (const extAttr of extAttrs) {
-        if (extAttr.name === "NamedConstructor" && typeof extAttr.rhs.value === "string") {
+        if (extAttr.name === "NamedConstructor" && extAttr.rhs && typeof extAttr.rhs.value === "string") {
             return {
                 name: extAttr.rhs.value,
                 signature: [{

